submissions/best-of-its-genre-nanogamer7/origins.md

```diff
@@ -0,0 +1,46 @@
+## Best of its genre – Nanogamer7
+
+We've all heard of Factorios insanely [good ratings](https://steamdb.info/stats/gameratings/) – consistently rising to second place after Portal 2 again and again.[^1] Obviously many others will try to imitate this success in the automation genre, but where did Factorio get inspired from? Was it the first game solely designed for automation or merely the first one to do it successfully? That's what I'll be trying to answer in this first article of hopefully a multi‐part series.
+
+### Automation defined
+
+The first difficult question we'll have to answer though, would be what automation even is. In Cities:Skylines for example you are building your city, which then (theoretically) functions autonomously. The difference here is, your job isn't to automate – the cars spawn out of nothing, choose their destination, and get there by themselves, once you build roads – but to manage it. In Minecraft on the other hand, you are the one to make the automation even possible – just with pistons and hoppers, instead of inserters and belts. Comparing these two examples, I think you can loosely define the automation genre as *"making stuff that makes stuff"*.
```

Comment:
> In Minecraft on the other hand, you are the one to make the automation even possible

I think you need to state that you're talking about vanilla here. It's still not that clear, although I'm struggling to think how to say it better. Maybe make an example, like in Factorio you use assembling machines, but in Vanilla Minecraft you need to build the assembling machine itself using various small dumb components.

Reply:
Hmm I thought about adding a screenshot of a vanilla smelting array, maybe that would make it more clear (not the best approach though)

Reply:
Ok I had another though about it, since I didn't mention mods earlier in the text I think it's fair to assume I'm talking about vanilla here. I could of course still add a small comment/bracket along the lines of "..., even in vanilla," or "In (vanilla) Minecraft..."

Reply:
I think it would be good enough to simply change it to 

> In Vanilla Minecraft on the other hand...

I think some people might assume that you're already talking about modded minecraft, because a lot of people think of those mods if you talk about automation in Minecraft - I think.